src/main/scala/org/broadinstitute/hail/utils/package.scala

```diff
@@ -0,0 +1,505 @@
+package org.broadinstitute.hail
+
+import java.io._
+import java.net.URI
+
+import org.apache.hadoop
+import org.apache.hadoop.fs.FileStatus
+import org.apache.hadoop.io.IOUtils._
+import org.apache.hadoop.io.compress.CompressionCodecFactory
+import org.apache.spark.AccumulableParam
+import org.broadinstitute.hail.check.Gen
+import org.broadinstitute.hail.io.compress.BGzipCodec
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.{TraversableOnce, mutable}
+import scala.io.Source
+import scala.language.{higherKinds, implicitConversions}
+import scala.reflect.ClassTag
+
+package object utils extends Logging
+  with richUtils.Implicits
+  with utils.NumericImplicits {
+
+  class FatalException(msg: String) extends RuntimeException(msg)
+
+  def info(msg: String) {
+    log.info(msg)
+    System.err.println("hail: info: " + msg)
+  }
+
+  def warn(msg: String) {
+    log.warn(msg)
+    System.err.println("hail: warning: " + msg)
+  }
+
+  def error(msg: String) {
+    log.error(msg)
+    System.err.println("hail: error: " + msg)
+  }
+
+  def fatal(msg: String): Nothing = {
+    throw new FatalException(msg)
+  }
+
+  def hadoopFS(filename: String, hConf: hadoop.conf.Configuration): hadoop.fs.FileSystem =
+    new hadoop.fs.Path(filename).getFileSystem(hConf)
+
+  private def hadoopCreate(filename: String, hConf: hadoop.conf.Configuration): OutputStream = {
```

Comment:
Could also put these on a RichHadoopConfiguration:
```
hConf.create(filename)
hConf.writeTextFile(filename) { ... }
```

that doesn't look terrible.